mteb/tasks/Image/ImageTextPairClassification/AROVisualAttribution.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import annotations
+
+from mteb.abstasks.Image.AbsTaskImageTextPairClassification import (
+    AbsTaskImageTextPairClassification,
+)
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class AROVisualAttribution(AbsTaskImageTextPairClassification):
+    images_column_names = ["image"]
+    texts_column_names = ["true_caption", "false_caption"]
+
+    metadata = TaskMetadata(
+        name="AROVisualAttribution",
+        description="Compositionality Evaluation of images to their captions.",
+        reference="https://openreview.net/forum?id=KRLUvxh8uaX",
+        dataset={
+            "path": "gowitheflow/ARO-Visual-Attribution",
+            "revision": "18f7e01358d91df599d723f00e16a18640e19398",
+        },
+        type="ImageTextPairClassification",
+        category="i2t",
+        eval_splits=["test"],
+        eval_langs=["eng-Latn"],
+        main_score="text_acc",
+        date=(
+            "2022-01-01",
+            "2022-12-31",
+        ),  # Estimated range for the collection of data
+        form=["written"],
+        domains=["Encyclopaedic"],
+        task_subtypes=["Caption Pairing"],
+        license="Not specified",
```

Comment:
really - no license?

Reply:
Hmm then we might accidentally have re-added it during the merge

Reply:
will investigate in the next main merge!

Reply:
> really - no license?

turns out I replied to the wrong comment yesterday. And yeah will update the license in the next commit.